src/org/pentaho/reporting/platform/plugin/output/CachingPageableHTMLOutput.java

```diff
@@ -221,6 +219,18 @@ public synchronized int generate( final MasterReport report, final int acceptedP
     }
   }
 
+  private int writeAllForScheduled( OutputStream outputStream, IReportContent cache ) throws IOException {
```

Comment:
@tmorgner I totally agree with you. But this is the way platform scheduler works for paginated html - it actually generates single page report. The difference is that in platform they choose output format before running the report, in our case the report is already running and we can't switch it to single page mode. I can't imagine the user that wants only one page of the report to be scheduled :)